lib/puppet/provider/reboot_notify/notify.rb

```diff
@@ -57,18 +65,22 @@ def self.post_resource_eval
       (current_time - v["updated"]) > Facter.value(:uptime_seconds)
     }
 
-    if not records.empty? then
+    unless records.empty?
       msg = ["System Reboot Required Because:"]
 
       records.each_pair do |k,v|
         # This is a fail safe for empty 'reasons'
-        records[k]['reason'] = 'modified' if ( records[k]['reason'].nil? or records[k]['reason'].empty? )
+        records[k]['reason'] = 'modified' if ( records[k]['reason'].nil? || records[k]['reason'].empty? )
```

Comment:
Why the change from 'or' to '||'?  They are equivalent and the first way is more understandable.

Reply:
Consistence with the rest of the codebase. We ended up moving to `||` from `or` across the board in the past in theory due to execution prioritization ordering.